services/venice-controller/src/main/java/com/linkedin/venice/controller/stats/ProtocolVersionAutoDetectionStats.java

```diff
@@ -17,13 +15,13 @@ public class ProtocolVersionAutoDetectionStats extends AbstractVeniceStats {
   public ProtocolVersionAutoDetectionStats(MetricsRepository metricsRepository, String name) {
     super(metricsRepository, name);
     protocolVersionAutoDetectionErrorSensor =
-        registerSensorIfAbsent(PROTOCOL_VERSION_AUTO_DETECTION_ERROR, new Count());
+        registerSensorIfAbsent(PROTOCOL_VERSION_AUTO_DETECTION_ERROR, new Gauge());
     protocolVersionAutoDetectionLatencySensor =
-        registerSensorIfAbsent(PROTOCOL_VERSION_AUTO_DETECTION_LATENCY, new Avg(), new Max());
+        registerSensorIfAbsent(PROTOCOL_VERSION_AUTO_DETECTION_LATENCY, new Gauge());
```

Comment:
Typo? We don't want to change latency metrics to Gauge, right?

Reply:
Updated to Avg, Thanks!